app/models/concerns/audit_associations.rb

```diff
@@ -8,6 +8,10 @@ def audited_attributes
       super.merge(associated_attributes)
     end
 
+    def previous_revision
```

Comment:
awesome addition, please also add apipie-dsl documentation for this method, I'm sure @ofedoren could give you pointers if needed

Reply:
Good point, missed it it's above.